data/json/mapgen/deserter_city/deserter_city_office.json

```diff
@@ -0,0 +1,318 @@
+        "s#?...s~~~~ss~~~~s..cc.#",
+        "s#O...s~~~~ss~~~~s.....#",
+        "s#h.O.s~~~~ss~~~~s.....#",
+        "s#^...sssss.sssss......#",
+        "s#h..O.................#",
+        "s#OO..O....DaF^........#",
+        "s#######..#####VGGVVVVV#",
+        "s#T#T#T#..#'''#.........",
+        "s#+#+#+#..#'''-.........",
+        "s#iiiii+..#'''#.........",
+        "s#CSCSC#..#####.........",
+        "s#######..+iyi#.........",
+        "s#CSCSC#..#####.........",
+        "s#iiiii+..#EEE#........#",
+        "s#+#+#+#..#EEE-....#####",
+        "s#T#T#T#.^#EEH#....+..<#",
+        "s#######=###############",
+        "ssssssssssssssssssssssss"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": { "u": "t_gutter_downspout" },
+      "place_npcs": [ { "class": "deserter", "x": 21, "y": 4 }, { "class": "deserter", "x": 17, "y": 17 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_2f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "/ugggg#####ww###wwww###`",
+        "/g````#...G..V.......f#`",
+        "/g````#hd.V..V..htth.fw`",
+        "/g````waf^V..V..htth..w`",
+        "/g````#####..G..htth..w`",
+        "/g````#...G..Va......^#`",
+        "/g````#hd.V..##########`",
+        "/g````waf.V..#.......a#`",
+        "/##########..#.ccccccc#`",
+        "/#fff#^......+........#`",
+        "/#...+.......#.....???#`",
+        "/########+#####VGGVVVV#`",
+        "/#T#T#T#yi#EEE#.......#`",
+        "/#+#+#+####EEE-......?w`",
+        "/#iiiii+..#EEH#......?#`",
+        "/#CSCSC#..#####......nw`",
+        "/#######.............?#`",
+        "/#CSCSC#..#####......?w`",
+        "/#iiiii+..#///#.......#`",
+        "/#+#+#+#..#///-..,##:##`",
+        "/#T#T#T#..#///#...#,.>#`",
+        "/#######ww#####ww,,#,##`",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": { "g": "t_gutter", "u": "t_gutter_drop", "`": "t_flat_roof", ":": "t_door_b" },
+      "place_npcs": [
+        { "class": "deserter", "x": 13, "y": 6 },
+        { "class": "deserter", "x": 9, "y": 19 },
+        { "class": "deserter", "x": 17, "y": 5 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_3f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "//////#################/",
+        "//////#iii*.......*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "//////#****...c...****#/",
+        "//////#iii*...Q...*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "/RRRRR#****...c...****#/",
+        "/Rssss#...............#/",
+        "/Rssss-...............#/",
+        "/*****############:####/",
+        "/*''''h''a#ZZZ#''x''''#/",
```

Comment:
Terrain issue under that broken console: should be t_floor, instead of t_linoleum_gray